src/routes/index.ts

```diff
@@ -339,7 +339,7 @@ router.route(config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL)
     navigation.hasOverseasEntity
   )
   .get(confirmOverseasEntityDetails.get)
-  .post(companyAuthentication, confirmOverseasEntityDetails.post);
+  .post(confirmOverseasEntityDetails.post);
```

Comment:
This page is before the company authentication. So should be ok to load all public data. But must not load any private data until after the auth code is entered. Need to cross check with other ticket.

Reply:
Good point, I'll raise that with them also